home.css

```
@@ -0,0 +1,226 @@
+@font-face {
+	font-family: 'Pretendard Variable';
+	font-weight: 45 920;
+	font-style: normal;
+	font-display: swap;
+	src: url('./src/PretendardVariable.woff2') format('woff2-variations');
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  font-family: 'Pretendard Variable';
+  flex-shrink: 0;
+}
+
+:root {
+  --click: #1251AA;
+  --brand: #3692FF;
+  --banner: #CFE5FF;
+  --coolgray-200: #E5E7EB;
+  --coolgray-400: #9CA3AF;
+  --coolgray-700: #374151;
+  --coolgray-900: #111827;
+  --error: #F74747;
+}
+
+
+  /* header */
+header {
+  height: 70px;
+  padding: 0 200px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo {
+  width: 153px;
+  height: 51px;
+}
+
+  /* banner */
+.banner-background {
+  background-color: var(--banner);
+  height: 540px;
+}
+
+.banner-container {
+  height: 540px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0 200px;
+}
+
+.banner-img {
+  max-height: 547px;
+  max-width: 996px;
+  width: 100%;
+  height: auto;
+  padding: 93px 0 0 ;
+  margin-top: auto;
+  margin-right: 200px;
+}
+
+img.banner-bottom {
+  max-height: 540px;
+  padding: 0;
+}
+
+.banner-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-left: 200px;
+  gap: 24px;
+  z-index: 1;
+}
+
+.banner-content > p {
```

```suggestion
.banner-content>p {
```
결합자와 선택자 사이에 공백이 있어도 의도한대로 적용은 되지만, 
css는 공백도 결합자로 취급해서 자식결합자는 공백 없이 사용하는게 좋겠습니다.